.github/workflows/main.yml

```diff
@@ -21,6 +21,20 @@ jobs:
         run: |
           bazel test //...
 
+  format:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup clang-format
+      run: |
+        wget https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.1/clang+llvm-10.0.1-x86_64-linux-gnu-ubuntu-16.04.tar.xz
+        tar -xvf clang+llvm-10.0.1-x86_64-linux-gnu-ubuntu-16.04.tar.xz
+        sudo mv ./clang+llvm-10.0.1-x86_64-linux-gnu-ubuntu-16.04/bin/clang-format /usr/local/bin
+        rm -rf clang+llvm-10.0.1-x86_64-linux-gnu-ubuntu-16.04/
+        git clone https://github.com/Sarcasm/run-clang-format.git
+    - name: Run clang-format
+      run: find ./ -iname "*.h" -o -iname "*.cc" | xargs ./run-clang-format/run-clang-format.py
```

Comment:
If the format is illegal, the CI will fail, right?

Reply:
Yes. we should follow specific format.